packages/grid/x-data-grid-generator/package.json

```diff
@@ -32,7 +32,7 @@
   "dependencies": {
     "@babel/runtime": "^7.22.6",
     "@mui/base": "^5.0.0-beta.7",
-    "@mui/x-data-grid-premium": "6.10.0",
+    "@mui/x-data-grid-premium": "6.10.1",
```

Comment:
We should have released the generator package as well because otherwise, you might end up using two versions on the premium package - see https://github.com/mui/mui-x/issues/9807#issuecomment-1653928455